Homework5/Task2/Collections/Catalog.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.Collections;
+using System.Text;
+using Task2.Classes;
+
+namespace Task2.Collections
+{
+ class Catalog : IEnumerable<(string, Book)>
+ {
+  private List<(string, Book)> _books = new List<(string, Book)> ();
+
+  private string ReturnNormalizedISBNFormat(string ISBN)
+  {
+   if (ISBN.Length == 17)
+   {
+    StringBuilder stringBuilder = new StringBuilder();
+    for (int i = 0; i < 17; i++)
+    {
+     if (i != 3 && i != 5 && i != 8 && i != 15)
+     {
+      stringBuilder.Append(ISBN[i]);
+     }
+    }
+    return stringBuilder.ToString();
+   }
+   return ISBN;
+  }
+
+  private bool IsDuplicate((string ISBN, Book book) bookTuple)
```

Comment:
typo IsDu**p**licate

Reply:
I believe "Duplicate" is the correct use of the word, unless you mean a different one, then I cant seem to spot which one.

Reply:
Yeah, I checked vocabulary and actually you are right, I just missed to delete this note.